go.mod

```diff
@@ -9,9 +9,17 @@ require (
 	github.com/hashicorp/go-uuid v1.0.3
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
+	github.com/itera-io/taikungoclient v0.0.0-20231024102014-2547653104f9
+	github.com/itera-io/taikungoclient/client v0.0.0-20231024102014-2547653104f9
 	github.com/robfig/cron/v3 v3.0.1
 )
 
+replace (
+	github.com/itera-io/taikungoclient => /home/radek/taikun/taikun-cli/taikungoclient-official
+	github.com/itera-io/taikungoclient/client => /home/radek/taikun/taikun-cli/taikungoclient-official/client
```

Comment:
Are you going to use clients that's stored in your laptop instead of the clients stored in github.com?